main.c

```diff
@@ -30,34 +31,44 @@ int main(int argc, char** argv)
     int runtime_s = 10;
     int ch;
     bool ttfb_only = false;
+    bool gso = false;
 
-    while ((ch = getopt(argc, argv, "p:sc:t:he")) != -1) {
+    while ((ch = getopt(argc, argv, "c:egp:st:h")) != -1) {
         switch (ch) {
```

Comment:
To keep the indentation consistent with the rest, we should keep the `case` statements aligned with the `switch` statement.

Reply:
Addressed in https://github.com/rbruenig/qperf/pull/5/commits/a339fb93d96eaecb8a5c456080b9fe3d3cd85cb5